stylesheets/components/Badge/Badges.css

```diff
@@ -0,0 +1,114 @@
+  /* Badge Container */
+  .badge__container {
+      position: relative;
+      width: max-content;
+  }
+/* Badge Types */
+  /* Default Badge */
+  .badge {
+      position: absolute;
+      top: -12px;
+      right: -12px;
+      font-size: 0.875rem;
+      font-weight: 500;
+      min-width: 24px;
+      min-height: 24px;
+      background-color: var(--primary-color);
+      color: var(--primary-text);
+      border-radius: 6px;
+  }
+
+  /* Badge on Icon */
+  .badge--icon {
+      top: -10px;
+      right: -10px;
+  }
+
+  /* Rounded Badge */
+  .badge--round {
+      border-radius: 50%;
+  }
+
+  /* Tag Badge */
+  .badge--tag {
+    position: absolute;
+    top: -12px;
+    right: -18px;
+    width: fit-content;
+    border-radius: 4px;
+    padding: 0px 6px;
+    font-size: 0.875rem;
+    text-transform: capitalize;
+}
+
+  /* Avatar Badge */
+  .badge--avatar {
+    background-color: var(--primary-color);
+    top: 0px;
+    right: 0px;
+    border: 3px solid var(--white);
+}
+
+
+  /* Badge Sizes */
+  /* Badge Large */
+  .badge--large {
+      min-width: 32px;
+      min-height: 32px;
+      top: -16px;
+      right: -16px;
+      font-size: 1rem;
+      font-weight: 800;
+  }
+
+  /* Badge Small */
+  .badge--small {
+      min-width: 16px;
+      min-height: 16px;
+      top: -8px;
+      right: -8px;
+      font-size: 0.625rem;
+      font-weight: 700;
+  }
+
+  /* Tag Badges Sizes */
+  /* Tag Badge Large */
+  .badge--tag--lg {
+      border-radius: 4px;
+      font-size: 1rem;
+      padding: 4px 8px;
+      top: -12px;
+      right: -30px;
+      background-color: var(--primary-color);
+      color: var(--primary-text);
+  }
+
+  /* Tag Badge Small */
+  .badge--tag--sm {
+      top: -10px;
+      right: -9px;
+      border-radius: 6px;
+      text-decoration: uppercase;
```

Comment:
Does text-decoration has this property?

Reply:
+1

Reply:
text transform*
text decoration is used to add over/underlines to the text .

Reply:
My Bad. It is Text Transform. Thank You  @coldpigli , @SimranjitKaurMaan , @Sameony . It is corrected in the latest commit.